internal/series/pointstores/elasticsearch_test.go

```diff
@@ -198,3 +214,25 @@ func TestLoadTestSet(t *testing.T) {
 		t.Fatalf("Failed to load test data (%s)", err.Error())
 	}
 }
+
+func TestMain(m *testing.M) {
+	// Setup
+	ctx := context.Background()
+	elastic, url, err := startElastic(ctx)
+	if err != nil {
+		fmt.Printf("Error starting test elasticsearch container (%s)", err.Error())
```

Comment:
Elasticsearch

Reply:
Fixed and corrected other occurrences in elasticsearch.go